localflavor/ch/forms.py

```diff
@@ -130,3 +132,54 @@ def clean(self, value):
             raise ValidationError(self.error_messages['invalid'])
 
         return '%s%s%s' % (idnumber, pos9, checksum)
+
+
+class CHSocialSecurityNumberField(forms.CharField):
```

Comment:
I thought about it but then based it on the the US-version (which is a `CharField` AFAIK).
Probably a `RegexField` plus a separate EAN-13 validation would do the trick.